hybrid/compartment_examples/inter_comp_call/secure-update_ddc/main.c

```diff
@@ -0,0 +1,172 @@
+/* This example attempts to modify the DDC of `switch_compartment`, assuming it
+ * knows the address where it resides (leaked via `c20` in `shared.S:21`, and
```

Comment:
That file/line number doesn't seem to exist.

Reply:
I can't seem to find where this comment comes from. I'd say it comes from an outdated version, but Github doesn't mark it as such? I don't think this is how it is in the most current version [1], but please point me in the right direction if I'm wrong.

[1] https://github.com/capablevms/cheri-examples/pull/57/files#diff-2adcbeed62c5d5cd6351228f9eb71fd19efe139c901677c406515c94b78a02ffR1-R2